modules/virt-viewing-network-state-of-node.adoc

```diff
@@ -26,7 +26,7 @@ $ oc get nns node01 -o yaml
 .Example output
 [source,yaml]
 ----
-apiVersion: nmstate.io/v1alpha1
+apiVersion: nmstate.io/v1beta1
```

Comment:
Same

Reply:
@phoracek indicated this was `nmstate.io/v1alpha1` above